environments/production/app-processes.yml

```diff
@@ -75,7 +75,8 @@ celery_processes:
       concurrency: 8
       max_tasks_per_child: 5
     saved_exports_queue:
-      concurrency: 6
+      num_workers: 3
+      concurrency: 2
```

Comment:
i am not super sure that this is how we should be adding this to have multiple workers on the same machine, so requesting reviewer to confirm that.

Reply:
Seems fine to me. It looks like the docs just say to test it out, they don't really give any hints on how to figure out the difference in performance of workers vs concurrency http://docs.celeryproject.org/en/3.1/userguide/workers.html#concurrency

Reply:
Thanks @emord . Is this g2g?

Reply:
Does num_workers vs. concurrency really mean much if it's not using gevent? I thought it didn't but I don't have a deep understanding

Reply:
one more thing i noticed that i we have concurrency of say, 6. No new tasks are taken until all 6 are completed, so if 5 tasks are done and 1 is still pending, the worker does not take any more tasks which is not a good behaviour.

Reply:
@dannyroberts from the docs i linked to:

> There is even some evidence to support that having multiple worker instances running, may perform better than having a single worker

I'm not sure why that is. it does seem confusing